storage/manyrocks/manyrocks.go

```diff
@@ -229,6 +230,77 @@ func (t *ManyRocks) OpenExtent(id s.ExtentUUID, keyPattern s.KeyPattern, notify
 	}, nil
 }
 
+// ListExtents returns a list of extents available on the store
+func (t *ManyRocks) ListExtents() (extents []s.ExtentUUID, err error) {
+
+	dir, err := os.Open(t.opts.BaseDir)
+	if err != nil {
+		return nil, err
+	}
+
+	defer dir.Close()
+
+	names, err := dir.Readdirnames(0)
+
+	if err == nil {
+
+		extents = make([]s.ExtentUUID, len(names))
+
+		for i, x := range names {
+			extents[i] = s.ExtentUUID(x)
+		}
+	}
+
+	return
+}
+
+// GetExtentInfo returns information about an extent
+func (t *ManyRocks) GetExtentInfo(id s.ExtentUUID) (info *s.ExtentInfo, err error) {
+
+	path := t.getDBPath(id) // get path to db
+
+	dir, err := os.Open(path)
+	if err != nil {
+		return nil, errOpenFailed
+	}
+
+	defer dir.Close()
+
+	dirStat, err := dir.Stat()
+
+	if err != nil {
+		return nil, errOpenFailed
+	}
+
+	modified := dirStat.ModTime().UnixNano() // get modified time
+
+	stat := dirStat.Sys().(*syscall.Stat_t)
```

Comment:
stat, ok := ... 

just to be safe.. this is a debug api.. don't want the process to crash fatal, if the type conversion fails.